shell/src/main/java/alluxio/cli/fs/command/LocationCommand.java

```diff
@@ -47,10 +47,10 @@ public String getCommandName() {
   }
 
   @Override
-  protected void runCommand(AlluxioURI path, CommandLine cl) throws AlluxioException, IOException {
-    URIStatus status = mFileSystem.getStatus(path);
+  protected void runPath(AlluxioURI plainPath) throws AlluxioException, IOException {
```

Comment:
actually how about calling this method `runPlainPath`?

Reply:
done